src/main.h

```diff
@@ -106,6 +106,10 @@ static const unsigned int AVG_INVENTORY_BROADCAST_INTERVAL = 5;
 static const unsigned int AVG_FEEFILTER_BROADCAST_INTERVAL = 10 * 60;
 /** Maximum feefilter broadcast delay after significant change. */
 static const unsigned int MAX_FEEFILTER_CHANGE_DELAY = 5 * 60;
+/** Block download timeout base, expressed in millionths of the block interval */
+static const int64_t BLOCK_DOWNLOAD_TIMEOUT_BASE = 2000000;
```

Comment:
Do I understand this correctly that `BLOCK_DOWNLOAD_TIMEOUT_BASE` is thus 2000000*(10min/1000000), ie. 20minutes? In either case, it is worth more expressive comment IMO.